Graphics/GraphicsEngineD3D11/src/ShaderD3D11Impl.cpp

```diff
@@ -54,12 +54,13 @@ static const ShaderVersion HLSLValidateShaderVersion(const ShaderVersion& Versio
 
 static const ShaderVersion GetD3D11ShaderModel(D3D_FEATURE_LEVEL d3dDeviceFeatureLevel, const ShaderVersion& HLSLVersion)
 {
-    switch (d3dDeviceFeatureLevel)
+    switch (static_cast<Uint32>(d3dDeviceFeatureLevel))
```

Comment:
I would suggest disabling this specific warning on MSVC in case 
`#ifndef NTDDI_WIN10_FE`

Reply:
If so, what's the point of fall back?